components/dropdown/src/floatingUI.mjs

```diff
@@ -10,22 +10,53 @@ export default class AuroFloatingUI {
     this.keyDownHandler = null;
   }
 
+  mirrorSize(fullscreen) {
+    // mirror the boxsize from bibSizer
+    const sizerStyle = window.getComputedStyle(this.element.bibSizer);
+    const bibContent = this.element.bib.shadowRoot.querySelector(".container");
+    if (fullscreen) {
+      bibContent.style.width = '';
+      bibContent.style.height = '';
+      bibContent.style.maxWidth = '';
+      bibContent.style.maxHeight = '';
+    } else {
+      if (sizerStyle.width !== '0px') {
+        bibContent.style.width = sizerStyle.width;
+      }
+      if (sizerStyle.height !== '0px') {
+        bibContent.style.height = sizerStyle.height;
+      }
+      bibContent.style.maxWidth = sizerStyle.maxWidth;
+      bibContent.style.maxHeight = sizerStyle.maxHeight;
+    }
+  }
+
+  getPositioningStrategy() {
+    if (this.element.bib.mobileFullscreenBreakpoint) {
+      const isMobile = window.matchMedia(`(max-width: ${this.element.bib.mobileFullscreenBreakpoint})`).matches;
+      return isMobile ? 'fullscreen' : 'floating';
+    }
+    return 'floating';
+  }
+
   position() {
+    const strategy = this.getPositioningStrategy();
+    if (strategy === 'fullscreen') {
+      this.handleMobileFullscreen(true);
+      this.mirrorSize(true);
+      return;
```

Comment:
We aren't actually returning anything in this function and the JSDoc syntax likely won't be happy once you add but have nothing to actually return in the code. I think it would be better if we use an `else {...}` here rather than a `return` to force an execution break.